documentation/src/main/asciidoc/topics/config_examples/hotrod-client-tls.properties

```diff
@@ -0,0 +1,15 @@
+infinispan.client.hotrod.server_list = 127.0.0.1:11222
+infinispan.client.hotrod.use_ssl = true
+infinispan.client.hotrod.sni_host_name = myservername
+# Keystore that contains the public keys for {brandname} Server.
+# Clients use the trust store to verify {brandname} Server identities.
+infinispan.client.hotrod.trust_store_file_name = server_truststore.pkcs12
+infinispan.client.hotrod.trust_store_password = changeme
+infinispan.client.hotrod.trust_store_type = PCKS12
+# Keystore that contains client certificates.
+# Clients present these certificates to {brandname} Server.
+infinispan.client.hotrod.key_store_file_name 	= client_keystore.pkcs12
```

Comment:
there is a "tab" hidden here
this is what I get in the HTML `infinispan.client.hotrod.key_store_file_name         = client_keystore.pkcs12`